org.jacoco.core/src/org/jacoco/core/internal/analysis/ClassAnalyzer.java

```diff
@@ -102,8 +102,7 @@ private boolean isMethodFiltered(final String className,
 				return true;
 			}
 		}
-		return (access & Opcodes.ACC_SYNTHETIC) != 0
```

Comment:
Deleting these lines needs more cleanup to get rid of compiler warnings:

- `import org.objectweb.asm.Opcodes;` is unused and should be deleted
- Method parameter `access` is unused and should be removed